src/components/emoji/nimbleEmoji.vue

```diff
@@ -25,97 +23,37 @@ export default {
     }
   },
   computed: {
+    view() {
+      return new EmojiView(
+          this.emoji, this.set, this.native, this.fallback,
+          this.size, this.sheetSize, this.backgroundImageFn)
+    },
     emojiData() {
-      return getData(this.emoji, this.skin, this.set, this.data)
+      return this.emoji._data
     },
     sanitizedData() {
-      return getSanitizedData(this.emoji, this.skin, this.set, this.data)
-    },
-    canRender() {
```

Comment:
Rendering logic moved to `EmojiView` helper which is reused to render emojis here, in a standalone emoji component and inside the category.